### PR TITLE
Add scheil recipe

### PR DESCRIPTION
**Pull request by bocklund**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/scheil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+
```
Comment (npavlovikj) [inline recipes/scheil/meta.yaml]: You need to add `script: "{{ PYTHON }} -m pip install . -vv"` here.

File: recipes/scheil/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "scheil" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "de43e05fb9fdfb084fa3c7709000cc81c88292eaee24a0ac020c52c032c0c809" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (npavlovikj) [inline recipes/scheil/meta.yaml]: Please remove the `fn` line - it is not necessary with the new conda version.

File: recipes/scheil/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
```
Comment (npavlovikj) [inline recipes/scheil/meta.yaml]: This should be `host` instead.

File: recipes/scheil/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+    - setuptools
```
Comment (npavlovikj) [inline recipes/scheil/meta.yaml]: Please add `pip` in the `host` requirements. Also, I don't think `setuptools` is needed here.

File: recipes/scheil/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - python
```
Comment (npavlovikj) [inline recipes/scheil/meta.yaml]: This should probably be `python >=3`.

**Comment by npavlovikj** [comment]
@bocklund , I left you few comments. Please see https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages for more information of the recipe structure.

**Comment by bocklund** [comment]
@npavlovikj thanks for your help. It looks like I have this working now. 

The Windows build is currently failing because a compatible version of one of the dependencies (`pycalphad`) is not yet available on Windows because of some other dependency issues that are currently being addressed (in particular: https://github.com/conda-forge/ipopt-feedstock/pull/34). 

This package is pure Python and in principle noarch and I can confirm that it works with a compatible version of pycalphad (we have a compatible version distributed through the "pycalphad" channel on Anaconda http://anaconda.org/pycalphad/pycalphad).

I'd prefer to keep this package as noarch, so everything will *just work* when pycalphad is fixed and released on conda-forge (for now, we are directing users to use the pycalphad channel as well). Can this be merged in the current state?

**Comment by moorepants** [comment]
It's probably best to disable the windows builds for now and then you can merge this. The windows builds can be enabled at a later date.

**Comment by bocklund** [comment]
@moorepants thanks for the advice. I removed `noarch: python` and added the skip for Windows and incompatible Python versions.

Who can merge this?

File: recipes/scheil/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+  skip: true  # [py<36]
```
Comment (scopatz) [inline recipes/scheil/meta.yaml]: Please make this `noarch: python`.   It really reduces the number of artifacts in the system

**Comment by bocklund** [comment]
@scopatz  According to the [docs](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python) I can’t use the selector

```yaml
build:
  skip: true  # [win] 
```

to not build on Windows, but the Windows build will fail because of a compatible version of all the dependencies is not available through conda-forge on Windows. 

What should I do?

**Comment by scopatz** [comment]
That is OK.  Noarch pachaes are only built on Linux

File: recipes/scheil/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A Scheil-Gulliver simulation tool using pycalphad'
+  doc_url: A Scheil-Gulliver simulation tool using pycalphad
+  dev_url: https://github.com/pycalphad/scheil
+
+extra:
+  recipe-maintainers:
+    - bocklund
+    - richardotis
```
Comment (scopatz) [inline recipes/scheil/meta.yaml]: @richardotis are you OK with being a maintainer?

Comment (richardotis) [reply recipes/scheil/meta.yaml]: Yes I consent to being a maintainer of this package

**Comment by bocklund** [comment]
Thank you for your help, @scopatz!

**Comment by scopatz** [comment]
My pleasure!

**Pull request merged**